src/models/spørsmal.tsx

```diff
@@ -0,0 +1,23 @@
+export enum IKategori {
+  Medlemskap = 'medlemskap',
+  Barn = 'barn',
+}
+
+export enum Medlemskap {
+  søkerBosattINorgeTreSisteÅr = 'søkerBosattINorgeTreSisteÅr',
+  søkerOppholderSegINorge = 'søkerOppholderSegINorge',
+  søkerErFlyktning = 'søkerErFlyktning',
+}
+
+export interface ISpørsmål {
+  spørsmål_id: string;
+  tekstid: string;
+  kategori: IKategori;
```

Comment:
Har lagt til kategori her for å kunne skille på spørsmål med den forutsetningen av vi skal ha alle spørsmål i en spørsmålconfig fil. Men vi kan også skille på spm ved å ha dem som egne const/ filer :)